inference-engine/src/inference_engine/os/win/win_shared_object_loader.cpp

```diff
@@ -129,8 +213,12 @@ class SharedObjectLoader::Impl {
 
     explicit Impl(const char* pluginName) {
         ExcludeCurrentDirectoryA();
+        LoadPluginFromDirectoryA(pluginName);
+
+        if (!shared_object) {
```

Comment:
will we have situations when `shared_object` is not loaded in `LoadPluginFromDirectoryA`?

Reply:
Theoretically  yes: 
#if !WINAPI_PARTITION_SYSTEM
        LoadSymbols();
        if (IEGetDllDirectoryA)

Reply:
ok. I see